setup.cfg

```diff
@@ -11,48 +9,50 @@ description = Community-developed python astronomy tools
 long_description = file: README.rst
 keywords = astronomy, astrophysics, cosmology, space, science,units, table, wcs, samp, coordinate, fits, modeling, models, fitting, ascii
 classifiers =
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: BSD License
-    Operating System :: OS Independent
-    Programming Language :: C
-    Programming Language :: Cython
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: Implementation :: CPython
-    Topic :: Scientific/Engineering :: Astronomy
-    Topic :: Scientific/Engineering :: Physics
+	Intended Audience :: Science/Research
+	License :: OSI Approved :: BSD License
+	Operating System :: OS Independent
+	Programming Language :: C
+	Programming Language :: Cython
+	Programming Language :: Python :: 3
+	Programming Language :: Python :: Implementation :: CPython
+	Topic :: Scientific/Engineering :: Astronomy
+	Topic :: Scientific/Engineering :: Physics
 
 [options]
+packages = find:
 requires = numpy
 zip_safe = False
 tests_require = pytest-astropy
-setup_requires = numpy>=1.16
+setup_requires = setuptools_scm
 install_requires = numpy>=1.16
 python_requires = >=3.6
 
 [options.entry_points]
 console_scripts =
-    fits2bitmap = astropy.visualization.scripts.fits2bitmap:main
-    fitscheck = astropy.io.fits.scripts.fitscheck:main
-    fitsdiff = astropy.io.fits.scripts.fitsdiff:main
-    fitsheader = astropy.io.fits.scripts.fitsheader:main
-    fitsinfo = astropy.io.fits.scripts.fitsinfo:main
-    samp_hub = astropy.samp.hub_script:hub_script
-    showtable = astropy.table.scripts.showtable:main
-    volint = astropy.io.votable.volint:main
-    wcslint = astropy.wcs.wcslint:main
+	fits2bitmap = astropy.visualization.scripts.fits2bitmap:main
```

Comment:
Same, please don't change indentation, just makes review harder!

Reply:
I'm not sure why this happened, but this has been fixed!